Sources/Classes/RSEventFilteringPlugin.m

```diff
@@ -0,0 +1,85 @@
+//
+//  RSEventFilteringPlugin.m
+//  Rudder
+//
+//  Created by Desu Sai Venkat on 17/01/22.
+//  Copyright © 2022 Rudder Labs India Pvt Ltd. All rights reserved.
+//
+
+#import "RSEventFilteringPlugin.h"
+
+NSString* const DISABLE = @"disable";
```

Comment:
You should move these to `RSConstants` Class.

Reply:
Sure, will do